src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
Usually (for instance in HttpFoundation or the WebLink component), we try to be as close as possible of the underlying standard. I would prefer `TelType` here. Doing this allows to do some nice tricks. For instance, if you want to use [mozilla-services/react-jsonschema-form](https://github.com/mozilla-services/react-jsonschema-form), it will be easier to automatically generate the JSON wanter by this lib from Symfony's metadata if you follow the standard (this libs use official HTML5 types).